roles/zuul/files/jobs/lore.yaml

```diff
@@ -6,6 +6,10 @@
       - zuul-git-prep
       - shell: |
           #!/bin/bash -ex
+          sudo apt-get install -yqq curl
+          curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash
```

Comment:
Right, but that's definitely a larger conversation. The point of our images (from what I can tell) is to be as bare as possible to test that your dependency installation is robust. Later on I'm sure we'll be adding images for different languages and toolsets, but node seems non-essential for our base image since it's only being used for linting one project.